### PR TITLE
Send requests

### PR DESCRIPTION
**Pull request by klikooo**
The virtual human can now send requests. The MunicipalityProtocol module was made by @levilime. 

User story: As a developer, I want my virtual human to send requests so that the municipality can deny  or accept them.

### CODE REVIEW

**Comment by MaxGroenenboom** [comment]
Tested and it works, I approve.

File: src/DUWOagent/actions.act2g [hunk 1]
```diff
@@ -33,3 +33,13 @@ define building_plan_demolish_polygon(MultiPolygon,Surface_or_Ground) with
 define building_plan_demolish(BuildingID) with
 	pre { }
 	post{ }
+	
+% Answer a popup with a date
+define popup_answer_with_date(MessageID, AnswerID, Date) with
+	pre { }
+	post{ }
+
+% Answe a popup
```
Comment (MarcianoJ) [inline src/DUWOagent/actions.act2g]: typo

**Comment by klikooo** [comment]
Fixed @MarcianoJ , please review

**Comment by MarcianoJ** [comment]
Approved

**Comment by Denpeer** [comment]
Approved

**Pull request merged**